src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/events/publishers/DataComplianceAwsEventPusher.java

```diff
@@ -38,41 +39,53 @@ public OffenderDeletionAwsEventPusher(
     }
 
     @Override
-    public void sendPendingDeletionEvent(final OffenderPendingDeletionEvent event) {
+    public void sendPendingDeletionEvent(final OffenderPendingDeletion event) {
 
         log.trace("Sending referral of offender pending deletion: {}", event.getOffenderIdDisplay());
 
         sqsClient.sendMessage(generatePendingDeletionRequest(event));
     }
 
     @Override
-    public void sendReferralCompleteEvent(final OffenderPendingDeletionReferralCompleteEvent event) {
+    public void sendReferralCompleteEvent(final OffenderPendingDeletionReferralComplete event) {
 
         log.trace("Sending process completed event for request: {}", event.getBatchId());
 
         sqsClient.sendMessage(generateReferralCompleteRequest(event));
     }
 
     @Override
-    public void sendDeletionCompleteEvent(final OffenderDeletionCompleteEvent event) {
+    public void sendDeletionCompleteEvent(final OffenderDeletionComplete event) {
 
         log.trace("Sending offender deletion complete event: {}", event.getOffenderIdDisplay());
 
         sqsClient.sendMessage(generateDeletionCompleteRequest(event));
     }
 
-    private SendMessageRequest generatePendingDeletionRequest(final OffenderPendingDeletionEvent event) {
+    @Override
+    public void sendDataDuplicateResult(DataDuplicateResult event) {
+
+        log.trace("Sending offender deletion complete event: {}", event.getOffenderIdDisplay());
```

Comment:
Looks like this should be logging for **duplicate** result and not **deletion**.